tensorflow/core/kernels/BUILD

```diff
@@ -517,6 +517,16 @@ cc_library(
     deps = ["//third_party/eigen3"],
 )
 
+cc_library(
```

Comment:
Would `cub_or_rocprim` be a better name?

Reply:
I think `gpu_prim_hdrs` better reflects the purpose of the package. However, I can rename it as long as it does not take another 1.5 months to review the PR after the change.

Reply:
Point taken, but I wasn't really able to work for the last couple of weeks, and I think the PR got stuck in limbo.

Reply:
It's not you personally, it's the entire system. I still have open PRs originally submitted in early January. Any change (however minor) seems to kick the PR back to the end of the queue and then it takes up to a month for it to be reviewed again. So I am wary of making any changes.

However, in this case, I had to fix buildifier formatting anyway, I can do the renaming as well if you like.

Reply:
> Any change (however minor) seems to kick the PR back to the end of the queue
I'm not sure that's how it works, usually once the reviewer is looking into it, the turnaround should be quite reasonable.

The system (kinda sorta) works when a correct reviewer is assigned (either manually, or by the triaging team). At this point, for AMD-specific changes it's probably either @chsigg or me (though I normally only review XLA-specific stuff, but I can also help with general reviews).